checks.sh

```diff
@@ -2,21 +2,22 @@
 
 set -e
 
-SRC=$(find src -name "*.cc")
+INC=$(find include -name "*.hh")
+SRC=$(find src \( -name "*.cc" -o -name "*.hh" \))
```

Comment:
I think we should probably raise an error if header files are found in src?

Reply:
I'll look into it

Reply:
Fixed